src/ml/neural_net/style_transfer/mps_style_transfer.h

```diff
@@ -13,6 +13,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+#ifdef DHAS_MACOS_10_15
```

Comment:
Should be `HAS_MACOS_10_15`, no `D` at the front

Reply:
Done!